### DISCUSSION TITLE
Set cropper size same as stencil

### DISCUSSION THREAD

**Discussion by keithlei01** [advanced-cropper/vue-advanced-cropper#190] [General]
I am trying to use fixed static cropper and would like to set the cropper and stencil size always the same. I tried to modify the css but failed to achieve it. Can anyone share how to do that? Thanks!

**Comment by Norserium** [comment]
@keithlei01, I assume you need to use the [`stencilSize`](https://norserium.github.io/vue-advanced-cropper/guides/advanced-recipes.html#fixed-stencil) property + set the fixed size for the cropper by css rules. [The example](https://codesandbox.io/s/vue-advanced-cropper-stencil-size-20dbst?file=/src/App.vue).

**Reply by keithlei01** [reply]
I reference the example and it works now! And it seems I have default-boundaries="fit" at the beginning which cause it not working. Thanks!
